src/styles/saas-list.less

```diff
@@ -67,26 +66,28 @@
       }
 
       .card-icon {
+        align-items: center;
+        display: flex;
         font-size: 24px;
         text-align: center;
 
+        img{
```

Comment:
Nit:  add space between "img" and "{".

Would be great to put this rule after the media queries rather than before since the media queries are for .card-icon and this is for a separate node.